src/oauth/signature.cr

```diff
@@ -19,7 +19,7 @@ struct OAuth::Signature
 
   def compute(request, tls, ts, nonce)
     base_string = base_string(request, tls, ts, nonce)
-    Base64.strict_encode(OpenSSL::HMAC.digest :sha1, key, base_string)
```

Comment:
There's no need to change this line

Reply:
I've changed this to met the request in https://github.com/crystal-lang/crystal/pull/6975#discussion_r226915346.

I was using symbols in my contribution, but [I was suggested to create an enum](https://github.com/crystal-lang/crystal/pull/6975#discussion_r226878167). Done that.

Then, I was suggested to share this enum with other code pieces. And that is why I changed this line.

Reply:
That's correct. However, there's an undocumented feature of the language where you can pass a symbol to a method that has an enum restriction, and the symbol will be automatically converted to the enum. That's why you can change **just this line** back to what it was before.